common/sensor/dev/intel_peci.c

```diff
@@ -0,0 +1,180 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "sensor.h"
+#include "pal.h"
+#include <drivers/peci.h>
+#include "hal_peci.h"
+#include <string.h>
+#include "intel_peci.h"
+
+#define READ_RTY 0x03
```

Comment:
I am assuming RTY is retry, but it is not clear.  In this #define and in the function names below, retry should be spelled out if that is what it is intended to do.